### PR TITLE
chore(site): fix Algolia Docsearch

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**
I think we should add sitemap because
- algolia [advises](https://community.algolia.com/docsearch/recommended-configuration.html) add sitemap
- SEO advises add sitemap

About `priority` in sitemap https://github.com/jekyll/jekyll-sitemap/issues/196#issuecomment-341798669

### CODE REVIEW

File: site/_layouts/docs.html [hunk 1]
```diff
@@ -50,14 +50,12 @@ <h3>{{ group.title }}</h3>
...
-    apiKey: '4cd7a76d4bc286ae69fe26182a8d4b18',
-    indexName: 'devextreme_reactive',
-    inputSelector: '#docsearch',
-    algoliaOptions: { 'facetFilters': ["techno:{{product_link_parts[1]}}", "tool:{{product_link_parts[2]}}"] },
-    debug: false
-  });
+<script
+  type="text/javascript"
+  src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"
+  onload="docsearch({ apiKey: '4cd7a76d4bc286ae69fe26182a8d4b18', indexName: 'devextreme_reactive', inputSelector: '#docsearch', algoliaOptions: { 'facetFilters': ['techno:{{product_link_parts[1]}}', 'tool: {{product_link_parts[2]}}'] }, debug: false })"
```
Comment (ushkal) [inline site/_layouts/docs.html]: Let's format it with newlines

File: site/_config.yml [hunk 2]
```diff
@@ -2,18 +2,25 @@ title: DevExtreme Reactive Components
 baseurl: "/devextreme-reactive"
 version: "X.X.X"
 
+url: "https://devexpress.github.io/devextreme-reactive/" # uses for configure sitemap
```
Comment (ushkal) [inline site/_config.yml]: uses -> used

Comment (MaximKudriavtsev) [reply site/_config.yml]: Yes

File: site/_layouts/docs.html [hunk 3]
```diff
@@ -50,14 +50,18 @@ <h3>{{ group.title }}</h3>
...
 
 <script src="{{site.baseurl}}{{page.demos_script_link}}/dist/index.js?v={{ site.time | date: '%s' }}"></script>
-<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
 {% assign product_link_parts = page.product_link | split: "/" %}
-<script type="text/javascript">
-  docsearch({
+<script
+  type="text/javascript"
+  src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"
+  onload="docsearch({
```
Comment (DmitryBogomolov) [inline site/_layouts/docs.html]: `onload` accepts a function. Does `docsearch(...)` returns a function? Or should it be something like this?

```html
onload="function() { docsearch(...); }"
```

**Pull request merged**